app/services/listing_index_service/data_types.rb

```diff
@@ -17,7 +17,7 @@ module ListingIndexService::DataTypes
     [:per_page, :to_integer, :mandatory, gte: 1],
     [:keywords, :string, :optional],
     [:categories, :array, :optional],
-    [:listing_shape_id, :fixnum, :optional],
+    [:listing_shape_id, :array, :optional],
```

Comment:
What happens now, how does the conversion work at sphinx_adapter? This changes the data type from number to a list of numbers so someone downstream must convert it or else it wouldn't work with sphinx because the sphinx content hasn't changed? Does it work with more than one number in the array?